### PR TITLE
Fix project creation invalid geometries bug

### PR DESCRIPTION
**Pull request by Hagellach37**
This relates to #363 and #421 

I also add some more details to the error message we send to slack #316 

I can't reproduce #363, but still think that the tests should cover it now.

### CODE REVIEW

File: mapswipe_workers/mapswipe_workers/project_types/tile_map_service_grid/project.py [hunk 1]
```diff
@@ -141,7 +164,8 @@ def validate_geometries(self):
             )
             raise CustomError(
                 f"Project is to large: {project_area} sqkm. "
```
Comment (laurentS) [inline mapswipe_workers/mapswipe_workers/project_types/tile_map_service_grid/project.py:166]: "too large" (an "o" is missing)

**Review by laurentS — APPROVED** [review]
Nice work, I like that you're providing guidance to the user in your errors!

File: mapswipe_workers/mapswipe_workers/utils/slack_helper.py [hunk 2]
```diff
@@ -6,7 +6,10 @@
 
 
 def send_slack_message(
-    message_type: MessageType, project_name: str, project_id: str = None
+    message_type: MessageType,
+    project_name: str,
+    project_id: str = None,
```
[the remark this answers is not in the document]
Comment (Hagellach37) [reply mapswipe_workers/mapswipe_workers/utils/slack_helper.py:11]: good one. We don't have mypy set up at the moment... but I will adjust and add another issue that we should add mypy to pre-commit

**Comment by Hagellach37** [comment]
Hey @Matthias-Schaub ,
the tests did fail again because of this issue I think: #416 

I will merge the code and reopen the issue. Let's see what we can do about it.

**Pull request merged**